base/utils/R/utils.R

```diff
@@ -207,21 +207,21 @@ zero.bounded.density <- function(x, bw = "SJ", n = 1001) {
 ##' @author David LeBauer, Alexey Shiklomanov
 summarize.result <- function(result) {
   ans1 <- result %>%
-    dplyr::filter(n == 1) %>%
+    dplyr::filter(.data$n == 1) %>%
     dplyr::group_by(.data$citation_id, .data$site_id, .data$trt_id,
                     .data$control, .data$greenhouse, .data$date, .data$time,
                     .data$cultivar_id, .data$specie_id, .data$name, .data$treatment_id) %>%
     dplyr::summarize( # stat must be computed first, before n and mean
-      statname = dplyr::if_else(length(n) == 1, "none", "SE"),
+      .data$statname = dplyr::if_else(length(n) == 1, "none", "SE"),
```

Comment:
```suggestion
      statname = dplyr::if_else(length(.data$n) == 1, "none", "SE"),
```

the `.data` verb isn't needed (nor allowed) on the left side of an assignment. If this is a surprise, recall that the reason for using `.data` is to avoid confusion about which scope a name is coming from. But here there's no ambiguity -- `foo = bar` always creates a new `foo` in the current scope whether or not one exists elsewhere.

Reply:
@infotroph thank you for pointing this out , is there any other fixes to be done here?

Reply:
@ashiklom beat me to it. Looks like you resolved these below.